taxcalc/functions.py

```diff
@@ -865,18 +863,14 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, c24517,
     c63100 = max(0., c63100)
     c63200 = max(0., c63000 - c63100)
     c09600 = c63200
-    c62100_everyone = c62100
```

Comment:
Sorry for the late comment, but the change here doesn't make much sense to me now. In the diagnostic table, we are reporting AMTI, number of AMT filers and AMT liability. For AMT filers and AMT liability, if I were not mistaken, we are only reporting filers who actually **owe** positive AMT liability. 

After the code line changes here, however, for AMTI, we are reporting the total amount that includes everyone. What we are doing here for AMTI doesn't seem to accord with our following calculation for # of filers and AMT liability. 

I thus have a few suggestions regarding how to deal with AMTI:


- Still adopt the code that zeros out AMTI, however, with slight modifications. In particular, 
```
 if c09600 == 0.:
       c60000 = 0.
       c62100 = 0.
```

- Change the name of AMTI column in our current diagnostic table to a new name: effective AMTI.